### PR TITLE
chore: test adding ci 2

### PR DESCRIPTION
**Pull request by yosef406**

### CODE REVIEW

**Comment by yosef406** [comment]
i just noticed that i didn't add a reviewer

**Review by EliasSalom — APPROVED** [review]

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please fix this comment and write CI better from this, **you can but you don't want to lose more time on this** 

why??

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+      - name: build node ubuntu
+        uses: actions/checkout@v3
+
+      - name: Use Node.js &{{matrix.node}}
+        uses: actions/setup-node@v3
+        with:
+          node: ${{matrix.node}}
+      - run: npm install    
+      - run: npm run build
```
[the remark this answers is not in the document]
Comment (yosef406) [reply .github/workflows/ci.yml:24]: this was a test in class i will change the code so that it is more correct

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        node: [14,16,18]
+
+    steps:
+      - name: build node ubuntu
+        uses: actions/checkout@v3
+
+      - name: Use Node.js &{{matrix.node}}
+        uses: actions/setup-node@v3
+        with:
+          node: ${{matrix.node}}
```
Comment (yosef406) [inline .github/workflows/ci.yml:22]: question: do i have to use `node-version: ${{matrix.node-version}}` or is `node: ${{matrix.node}}` acceptable

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yosef406) [reply .github/workflows/ci.yml:34]: added the name for every run

File: .github/workflows/ci.yml (hunk 2 above)
Comment (yosef406) [reply .github/workflows/ci.yml]: i added `node-version` in stead of `node`

**Review by EliasSalom — APPROVED** [review]
LGTM

**Pull request merged**